### PR TITLE
feat(azure-pg): add drop-autodevops-dbs job

### PR DESCRIPTION
**Pull request by revolunet**
So we can use it in the CLI

### CODE REVIEW

File: src/components/azure-pg/__snapshots__/drop-autodevops-dbs.test.ts.snap [hunk 1]
```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should use default secret for running drop-autodevops-dbs 1`] = `
+Object {
+  "apiVersion": "batch/v1",
+  "kind": "Job",
+  "metadata": Object {
+    "name": "drop-azure-autodevops-dbs-undefined",
```
Comment (douglasduteil) [inline src/components/azure-pg/__snapshots__/drop-autodevops-dbs.test.ts.snap:8]: `undefined` ?

File: src/components/azure-pg/__snapshots__/drop-autodevops-dbs.test.ts.snap [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  },
+}
+`;
+
+exports[`should use some-secret for running drop-autodevops-dbs 1`] = `
+Object {
+  "apiVersion": "batch/v1",
+  "kind": "Job",
+  "metadata": Object {
+    "name": "drop-azure-autodevops-dbs-undefined",
```
Comment (douglasduteil) [inline src/components/azure-pg/__snapshots__/drop-autodevops-dbs.test.ts.snap:54]: `undefined` ?

File: src/components/azure-pg/drop-autodevops-dbs.test.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```
Comment (douglasduteil) [inline src/components/azure-pg/drop-autodevops-dbs.test.ts:1]: why ?

Comment (revolunet) [reply src/components/azure-pg/drop-autodevops-dbs.test.ts:1]: old code

**Comment by revolunet** [comment]
hop

**Review by douglasduteil — APPROVED** [review]

**Comment by revolunet** [comment]
thanks for review !

**Pull request merged**